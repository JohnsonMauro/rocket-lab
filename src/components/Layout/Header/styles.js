import styled from "@emotion/styled";

const HeaderStyles = styled.header`
  display: flex;
  height: 78px;
  justify-content: center;
  margin: auto 0;
`;

const HeaderImg = styled.img`
  width: 212px;
`;

export { HeaderStyles, HeaderImg };
