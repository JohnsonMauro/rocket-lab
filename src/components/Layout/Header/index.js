import { HeaderStyles, HeaderImg } from './styles';

const Header = () => (
  <HeaderStyles>
    <HeaderImg
      src="/static/public/images/crunchaccounting.png"
      alt="Crunch Accounting Logo"
    ></HeaderImg>
  </HeaderStyles>
);

export default Header;
