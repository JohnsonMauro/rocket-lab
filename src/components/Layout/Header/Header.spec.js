import React from 'react';
import { render } from '@testing-library/react';

import GlobalsContainer from '@test/GlobalsContainer';
import Header from './index';

describe('<Header />', () => {
  it('Renders the Header component', () => {
    const { container } = render(<Header />, {
      wrapper: GlobalsContainer,
    });
    expect(container).toMatchSnapshot();
  });
});
