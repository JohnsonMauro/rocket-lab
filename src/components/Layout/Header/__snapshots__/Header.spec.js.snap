// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Renders the Header component 1`] = `
.emotion-2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 78px;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin: auto 0;
}

.emotion-0 {
  width: 212px;
}

<div>
  <header
    class="emotion-2 emotion-3"
  >
    <img
      alt="Crunch Accounting Logo"
      class="emotion-0 emotion-1"
      src="mocked-file"
    />
  </header>
</div>
`;
